src/platform/EFR32/wifi/ethernetif.cpp

```diff
@@ -61,6 +61,8 @@ StaticSemaphore_t xEthernetIfSemaBuffer;
 
 #define LWIP_FRAME_ALIGNMENT 60
 
+uint32_t overrun_count = 0;
```

Comment:
since this is a global, should it be (a) `static` and (b) named as such like `gOverrunCount` or similar?

Same question for the above xEthernetIfSemaBuffer, but at least that has some x prefix and is less generic than a `overrun_count` (overrun of what?)

Reply:
Thanks for your input , I have made the changes accordingly.